.env.example

```
@@ -1,369 +1,122 @@
+
+# =============================================================================
+# WordPress/WooCommerce
+# =============================================================================
 WORDPRESS_URL=https://your-site.com
 WORDPRESS_USERNAME=admin
-WORDPRESS_PASSWORD=your-app-password
-WORDPRESS_APP_PASSWORD=your-app-password
-WORDPRESS_API_KEY=your-api-key
-
-# Get from: WooCommerce → Settings → Advanced → REST API
+WORDPRESS_APP_PASSWORD=xxxx-xxxx-xxxx-xxxx
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WORDPRESS_APP_PASSWORD key should go before the WORDPRESS_URL key

(UnorderedKey)